File: data-client/src/main/java/org/wikipedia/login/LoginResetPasswordResult.java
package org.wikipedia.login;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import org.wikipedia.dataclient.WikiSite;

public class LoginResetPasswordResult extends LoginResult {
    public LoginResetPasswordResult(@NonNull WikiSite site, @NonNull String status, @Nullable String userName,
                             @Nullable String password, @Nullable String message) {
        super(site, status, userName, password, message);
    }
}

File: app/src/main/java/fr/free/nrw/commons/auth/login/LoginResetPasswordResult.kt
package fr.free.nrw.commons.auth.login

import org.wikipedia.dataclient.WikiSite

class LoginResetPasswordResult(
    site: WikiSite,
    status: String,
    userName: String?,
    password: String?,
    message: String?
) : LoginResult(site, status, userName, password, message)
